CHANGELOG.md

```diff
@@ -0,0 +1,515 @@
+# Changelog
+
+## [Unreleased](https://github.com/denysdovhan/spaceship-prompt/tree/HEAD)
+
+[Full Changelog](https://github.com/denysdovhan/spaceship-prompt/compare/v3.0.2...HEAD)
+
+**Fixed bugs:**
+
+- \(\#q\) glob qualifier doesn't work for zsh 5.0.5 [\#190](https://github.com/denysdovhan/spaceship-prompt/issues/190)
+
+## [v3.0.2](https://github.com/denysdovhan/spaceship-prompt/tree/v3.0.2) (2018-01-18)
+
+[Full Changelog](https://github.com/denysdovhan/spaceship-prompt/compare/v3.0.1...v3.0.2)
+
+## [v3.0.1](https://github.com/denysdovhan/spaceship-prompt/tree/v3.0.1) (2018-01-18)
+
+[Full Changelog](https://github.com/denysdovhan/spaceship-prompt/compare/v3.0.0...v3.0.1)
+
+**Breaking changes:**
+
+- 3.0: Roadmap [\#200](https://github.com/denysdovhan/spaceship-prompt/issues/200)
+
+**Merged pull requests:**
+
+- Promote @maximbaz to collaborators [\#320](https://github.com/denysdovhan/spaceship-prompt/pull/320) ([denysdovhan](https://github.com/denysdovhan))
+
+## [v3.0.0](https://github.com/denysdovhan/spaceship-prompt/tree/v3.0.0) (2018-01-18)
+
+[Full Changelog](https://github.com/denysdovhan/spaceship-prompt/compare/v2.11.0...v3.0.0)
+
+**Breaking changes:**
+
+- 3.0 [\#148](https://github.com/denysdovhan/spaceship-prompt/pull/148) ([denysdovhan](https://github.com/denysdovhan))
+
+**New features:**
+
+- Make showing battery on 100% configurable [\#306](https://github.com/denysdovhan/spaceship-prompt/pull/306) ([maximbaz](https://github.com/maximbaz))
+
+**Merged pull requests:**
+
+- Make dir section a little faster when SPACESHIP\_DIR\_TRUNC\_REPO == false [\#312](https://github.com/denysdovhan/spaceship-prompt/pull/312) ([maximbaz](https://github.com/maximbaz))
+- Add zsh framework to issue template [\#310](https://github.com/denysdovhan/spaceship-prompt/pull/310) ([salmanulfarzy](https://github.com/salmanulfarzy))
+- Revert "Show 100% charge by default" [\#308](https://github.com/denysdovhan/spaceship-prompt/pull/308) ([maximbaz](https://github.com/maximbaz))
+-  Update options documentation for 3.0  [\#294](https://github.com/denysdovhan/spaceship-prompt/pull/294) ([salmanulfarzy](https://github.com/salmanulfarzy))
+- Drop LS\_COLORS variable [\#273](https://github.com/denysdovhan/spaceship-prompt/pull/273) ([maximbaz](https://github.com/maximbaz))
+
+## [v2.11.0](https://github.com/denysdovhan/spaceship-prompt/tree/v2.11.0) (2018-01-12)
```

Comment:
That's not true. Almost all of these features are parts of `3.0.0`

Reply:
Nicely spotted, it looks like the tool doesn't support parallel release branches, it looks at the merged time of a PR and matches it with the next closest release - and since there were bugfix releases in v2, it matched those PRs to them 😞 

Quick search reveals that this is an issue for others too: https://github.com/skywinder/github-changelog-generator/issues/580#issuecomment-338932413

@salmanulfarzy looks like we won't be able to use this generator.

Reply:
Toyed with some generators most of them share this. Since releases are not that frequent, Can we at least use this as basis for release notes and proceed with manual verification ?

It does provide *most* of the information correctly. It's really difficult to follow when there are no release notes/chage logs.

Reply:
I fully agree with the sentiment on the absence of changelog